docs/Refund.md

```diff
@@ -6,28 +6,64 @@ If you are looking for information about requesting a refund from a merchant, pl
 ## Create a refund
 
-1. After an invoice has been confirmed, go in the Invoices page and click `Details` on the invoice.
+1. After an invoice has been confirmed but needs a refund, go in the Invoices page and click `Details` on the invoice.
 
-![BTCPay Server refund feature](./img/pull-payments/9.png "BTCPay Server refund feature")
+![BTCPay Server refund feature](./img/refunds/invoices-details.jpg "BTCPay Server refund feature")
 
 2. Click `Issue a refund`
 
-![BTCPay Server refund feature](./img/pull-payments/10.png "BTCPay Server refund feature")
+![BTCPay Server refund feature](./img/refunds/issue-refund.jpg "BTCPay Server refund feature")
+
+3. Select the way how you want to issue the refund
+
+![BTCPay Server refund feature](./img/refunds/issue-refund-payment-option.jpg "BTCPay Server refund feature")
+
+4. Select the `amount` you want to refund
+
+![BTCPay Server refund feature](./img/refunds/issue-refund-amount.jpg "BTCPay Server refund feature")
+
+5. Share the link of this page with your customer
+
+![BTCPay Server refund feature](./img/refunds/claimingside.jpg "BTCPay Server refund feature")
+
+
+## Processing refund 
+
+In the next few steps, you follow along on processing a refund. 
+Costumer has claimed the refund, and this is how to process it.
+
+1. Goto the Payouts tab in your menu. 
+
+![BTCPay Server Payouts tab](./img/refunds/payouts-status3-options-appr.jpg "BTCPay Server refund feature")
+
+2. Select the Payouts you want to process, go to actions and select `Approve and send`
+
+![BTCPay Server Payouts tab](./img/refunds/payouts-status3-options-appr.jpg "BTCPay Server refund feature")
+
+3. Sign and broadcast the transactions.
+
+![BTCPay Server Payouts tab](./img/refunds/payouts-status4-options-sign3-adv.jpg "BTCPay Server refund feature")
+
+4. Payout has now been Signed and is in progress, this is reflected to the claimant in their view. 
+
+![BTCPay Server Payouts tab](./img/refunds/payout-status-succesfull.jpg "BTCPay Server refund feature")
+
+5. After the Transaction has been confirmed, the status of the Payout will be completed. 
+
+![BTCPay Server Payouts tab](./img/refunds/payouts-status5-completed1.jpg "BTCPay Server refund feature")
+
+Costumers view after the Refund has been paid.
 
-3. Select the way you want to issue the refund
+![BTCPay Server Payouts tab](./img/refunds/claiment-completed1.jpg "BTCPay Server refund feature")
 
-![BTCPay Server refund feature](./img/pull-payments/11.png "BTCPay Server refund feature")
 
-4. Share the pull payment link with the customer
+## Questions
 
-![BTCPay Server refund feature](./img/pull-payments/12.png "BTCPay Server refund feature")
+If you have questions about the BTCPay Server refunds function, you can join the [community chat](https://chat.btcpayserver.org/).
+If you have questions about any of the other tools or commands, etc., it's likely you can find answers to your questions by doing a search on the internet or on [StackOverflow](https://stackoverflow.com/).
```

Comment:
In the future, we may add this for every page in the footer, I just don't think we should add this to this specific page and not the others.
```suggestion
```